Library/Formula/tesseract.rb

```diff
@@ -58,8 +70,16 @@ def install
     ENV.cxx11
 
     system "./autogen.sh" if build.head?
-    system "./configure", "--disable-dependency-tracking", "--prefix=#{prefix}"
+    if build.with? "opencl"
```

Comment:
Just use an `args` array here, rather than duplicating the whole block.

```ruby
args = %W[
  --disable-dependency-tracking
  --prefix=#{prefix}
]

args << "--enable-opencl" if build.with? "opencl"

system "./configure", *args
```